simtools/model/camera.py

```diff
@@ -4,7 +4,7 @@
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
 import matplotlib.colors as mcolors
-import scipy.spatial.cKDTree as KDTree
+from scipy.spatial import cKDTree as KDTree
```

Comment:
That was the bug that I added and fixed later. It was fixed in the master, though.

Reply:
Yes, but when I fixed it in my branch and then asked to pull it wanted me to commit first etc. and I was lazy, so I figure the merge would take care of it instead.